documentation/docs/30-advanced/20-hooks.md

```diff
@@ -66,7 +66,7 @@ export async function handle({ event, resolve }) {
 }
 ```
 
-You can add call multiple `handle` functions with [the `sequence` helper function](/docs/modules#sveltejs-kit-hooks).
+You can define multiple `handle` functions and execute them in a middleware-like pattern with [the `sequence` helper function](/docs/modules#sveltejs-kit-hooks).
```

Comment:
i'm going to take out the reference to middleware for people who aren't familiar with middleware
```suggestion
You can define multiple `handle` functions and execute them with [the `sequence` helper function](/docs/modules#sveltejs-kit-hooks).
```